fix: more enhancement to UI

AceTheCreator opened the pull request:
I added a few enhancements based on suggested feedback

- Converted all used image formats from other formats to webp
- Fix duplicate images 
- Removed unused code 
- Created community components to hold the necessary component
- Added more community highlights to the community homepage

cc @derberg @alequetzalli @akshatnema 

#917

AceTheCreator committed:
Merge branch 'master' of https://github.com/AceTheCreator/website into community

AceTheCreator committed:
Merge branch 'community' of https://github.com/AceTheCreator/website into community

AceTheCreator committed:
Merge branch 'community' of https://github.com/AceTheCreator/website into community

AceTheCreator committed:
replaced image format

AceTheCreator committed:
removed unused component

AceTheCreator committed:
moved ambassadors content to a new file

AceTheCreator committed:
created components for community section

AceTheCreator committed:
changes implemented

AceTheCreator committed:
.

akshatnema commented on components/community/Card.js:
```diff
@@ -0,0 +1,69 @@
+        <Paragraph textColor={bg && 'text-gray-400'}>{description}</Paragraph>
+      </div>
+      <div className="mt-10">
+        <Link href={link}>
```
There should be a `<a>` tag below this `Link` component.

AceTheCreator replied:
Any reason why?

akshatnema replied:
The browser isn't able to recognize it as a link when you right-click on it.

AceTheCreator replied:
I see

akshatnema reviewed (changes requested):
![image](https://user-images.githubusercontent.com/76521428/223401842-4ce5fa30-40a7-415a-8cf9-6fa5221d9d31.png)

Please add some shadows to these cards as they look slightly odd from all the cards you made above.

akshatnema reviewed (commented):
![image](https://user-images.githubusercontent.com/76521428/223403503-0b55e7f9-3d7c-47fa-bab8-d082b9ec4174.png)

Please make this heading in TSC on `text-2xl` only.

AceTheCreator committed:
added shadow to cards and bd color

AceTheCreator commented:
> ![image](https://user-images.githubusercontent.com/76521428/223403503-0b55e7f9-3d7c-47fa-bab8-d082b9ec4174.png)
> 
> Please make this heading in TSC on `text-2xl` only.

I don't it's related to this PR :)

AceTheCreator commented:
@derberg, you might wanna take a look at this PR... It addresses your comment here --> `There is one problem I have with whatever I see when going to /community. There are highlights only of items that you added in this PR. But what about TSC? contributor dashboard? newsroom?`

derberg commented:
I love it all, thanks so much for taking my feedback into account 🙏🏼

quetzalliwrites commented on pages/community/index.js:
```diff
@@ -110,225 +47,149 @@ function CommunityIndexPage() {
+            bg="bg-code-editor-dark"
+            heading="Explore and Contribute to Hot Issues"
+            description="Discover over 100s of interesting issues, suitable for both new
+          contributors and existing contributors."
```
```suggestion
            description="Discover over 100s of interesting issues, suitable for new and existing contributors."
```

quetzalliwrites commented on pages/community/index.js:
```diff
@@ -110,225 +47,149 @@ function CommunityIndexPage() {
+            <Card
+              type="small"
+              taglineBg=""
+              tagline="Rockstars"
```
```suggestion
              tagline="Contributors"
```

quetzalliwrites commented on pages/community/index.js:
```diff
@@ -110,225 +47,149 @@ function CommunityIndexPage() {
-            </Heading>
+            <Card
+              type="small"
+              tagline="Community"
```
```suggestion
              tagline="Goals"
```

🤔  Isn't goals a more appropriate tagline since this card is about the community **goals**?

And so, because of that, I will also recommend you switch the emojis in this and the TSC card.

quetzalliwrites commented on pages/community/index.js:
```diff
@@ -110,225 +47,149 @@ function CommunityIndexPage() {
+            <Card
+              type="small"
+              tagline="Community"
+              icon="🚀"
```
```suggestion
              icon="🎯"
```

quetzalliwrites commented on pages/community/index.js:
```diff
@@ -110,225 +47,149 @@ function CommunityIndexPage() {
+            heading="Meet Folks that are Redefining the Initiative"
+            description="Learn how to become a technical steering committee member and see our current tsc members."
+            tagline="Projection"
+            icon="🎯"
```
```suggestion
            icon="🚀"
```

quetzalliwrites commented on pages/community/index.js:
```diff
@@ -110,225 +47,149 @@ function CommunityIndexPage() {
+            taglineBg="bg-primary-200"
+            heading="Meet Folks that are Redefining the Initiative"
+            description="Learn how to become a technical steering committee member and see our current tsc members."
+            tagline="Projection"
```
```suggestion
            tagline="TSC"
```

Yeah... Projection doesn't really say anything to me here, I think we should be clear and direct.

quetzalliwrites commented on pages/community/index.js:
```diff
@@ -110,225 +47,149 @@ function CommunityIndexPage() {
+        <div className="md:w-[50%] mt-10 md:mt-0">
+          <Card
+            taglineBg="bg-primary-200"
+            heading="Meet Folks that are Redefining the Initiative"
```
```suggestion
            heading="Meet Folks Redefining the Initiative"
```

quetzalliwrites commented on pages/community/index.js:
```diff
@@ -110,225 +47,149 @@ function CommunityIndexPage() {
+          <Card
+            taglineBg="bg-primary-200"
+            heading="Meet Folks that are Redefining the Initiative"
+            description="Learn how to become a technical steering committee member and see our current tsc members."
```
```suggestion
            description="Learn how to become a Technical Steering Committee  (TSC) member and see our current members."
```

quetzalliwrites reviewed (changes requested):
review done! ✅ ✨✨✨💪🏿

quetzalliwrites commented:
Wow, I love this iteration; these updates make this page extremely informative!!! 

THANK YOU, @AceTheCreator 💪🏿✨✨✨

AceTheCreator committed:
added suggested changes

AceTheCreator commented:
Added suggested changes cc @derberg @akshatnema @alequetzalli

AceTheCreator committed:
.

AceTheCreator commented:
/rtm

akshatnema commented:
@AceTheCreator You need to wait for @alequetzalli to approve the changes :smile:

AceTheCreator commented:
> @AceTheCreator You need to wait for @alequetzalli to approve the changes 😄

While you're right, I don't think she has any changes to suggest again 🙂

We need to speed up merging the community branch, so this pr is rtm

quetzalliwrites commented:
/rtm

quetzalliwrites commented:
> "We need to speed up merging... "

welcome to oss 😄
